### DISCUSSION TITLE
info Termostati

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#3284] [GENERALE - Discussioni generiche]
Ciao a tutti, vorrei sostituire i miei termostati per poterli integrare in HA. attualmente ho dei semplici termostati da incasso con scatola 503, mi stavo affacciando ai TADO (versione cablata) e notavo la nuova serie X. si può integrare senza problemi tramite l'app Tado e l'apposito gateway oppure lavorando con protocolli matter si può usare solo il termostato senza gateway?
altre info: considerato che la mia scatola da incasso è una 503 (3posti classici) esistono placche compatibili con TAdo?
altri temrostati sa consigliare escludendo TADO?

grazie a tutti in anticipo!

**Comment by indomus** [comment]
devi avere un thread border router per attivarlo presso l'app Tado, che sia "il suo" o uno che hai in rete a casa. ancora non sappiamo se l'integrazione classica via cloud funzioni o meno, sulla X. di certo puoi integrare via Matter (ovviamente coi limiti intrinsechi alle definizioni matter stesse).

**Comment by Leo69Leo** [comment]
Buongiorno, anche io ho idea di sostituire gli attuali termostati che ho in casa: ho un'abitazione su tre livelli, e - in ogni livello - è presente un cronotermostato - alimentato a batterie - che agisce e comanda una valvola di zona (quindi NON la caldaia), in modo da poter avviare la caldaia indipendentemente, tra un livello e l'altro (vi sono quindi, in totale, tre cronotermostati e tre valvole di zona). Stavo pensando alla soluzione TADO:
sapete dirmi
1 - se il cronotermostato intelligente TADO X permetta di comandare le valvole di zona (quindi NON direttamente la caldaia con "contatto pulito"; in breve - attualmente - ogni cronotermostato ha tre "connessioni": L + L1 cioè NO + L2 cioè NC);
2 - se sia possibile (con un unico account della relativa App) gestire tre cronotermostati, situati nella stessa abitazione?
Grazie mille

**Reply by Topastrodfogna** [reply]
Ciao,
la mia esperienza è simile alla tua, ma con la versione TADO classica.
Ho tre termostati cablati che gestiscono altrettante valvole di zona e dall'app funziana tutti tranquillamente.
Attualmente nella mia abitazione sono presenti 3 termostati cablati,  2 sensori di temperatura wireless, 7 teste termostatiche e 1 internet bridge.
Tutto e visibile e gestibile su HA tramite la relativa integrazione
